src/back/gql/models/users/types.js

```diff
@@ -16,6 +16,9 @@ export const UserType = new GraphQLObjectType({
     email: {
```

Comment:
Shouldn't this be unique? or is it not possible to evaluate uniqueness within GQL?

Reply:
Not possible perse, but the mutation will make sure to handle this :+1: When entering a value, I'll make sure you can't enter an existing one. This once we get into mutations pieces.